CMakeLists.txt

```diff
@@ -293,9 +293,12 @@ if(WIN32)
     # See https://gcc.gnu.org/bugzilla/show_bug.cgi?id=54412.
     try_append_cxx_flags("-Wa,-muse-unaligned-vector-move" TARGET core_interface SKIP_LINK)
     try_append_linker_flag("-static" TARGET core_interface)
-    # We require Windows 7 (NT 6.1) or later.
+    # We support Windows 10+, however it's not possible to set these values accordingly,
+    # due to a bug in mingw-w64 when using CRT. See https://sourceforge.net/p/mingw-w64/bugs/968/.
+    # As a best effort, set these values to target Windows 8, until we switch to building
+    # against UCRT.
```

Comment:
> however it's not possible to set these values accordingly, due to a bug in mingw-w64 when using CRT

I think blaming it on the CERT is incorrect? See [comment](https://github.com/bitcoin/bitcoin/pull/31172#issuecomment-2448647985):

> I was mistaken above, linking against UCRT doesn't make the executable build with /GS enabled.

Reply:
I've just shortened the comment here.